lib/omnibus-ctl.rb

```diff
@@ -506,6 +508,45 @@ def reconfigure(exit_on_success=true)
       end
     end
 
+    def check_license_acceptance(override_accept = false)
+      license_guard_file_path = File.join(etc_path, ".lic.accepted")
```

Comment:
Note to self: If this is put in /var/opt we may need to take special care that it gets backed up correctly for tools that provide a backup command.